ext/pdo_pgsql/tests/common.phpt

```diff
@@ -18,7 +18,7 @@ if (false !== getenv('PDO_PGSQL_TEST_DSN')) {
 		$config['ENV']['PDOTEST_ATTR'] = getenv('PDO_PGSQL_TEST_ATTR');
 	}
 } else {
-	$config['ENV']['PDOTEST_DSN'] = 'pgsql:host=localhost port=5432 dbname=test user= password=';
+	$config['ENV']['PDOTEST_DSN'] = 'pgsql:host=localhost port=5432 dbname=test user=postgres password=postgres';
```

Comment:
Agreed, but I had to because for some reason tests wasn't running because the PDOTEST complaint about no username and password informed. Why? I've actually didn't go further. I was worried more about the test not running than this line. 

My mistake. I've committed accidentally.